### PR TITLE
Extract blinding pubkey from confidential address

### PR DESCRIPTION
**Pull request by altafan**
This closes [#15](https://github.com/vulpemventures/liquid-js-monorepo/issues/15)

### CODE REVIEW

**Comment by tiero** [comment]
@altafan Solve conflicts

File: test/address.spec.test.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import * as assert from 'assert';
```
Comment (tiero) [inline test/address.spec.test.ts]: I would call it just address.spec.ts

File: src/address.js [hunk 2]
```diff
@@ -7,6 +7,19 @@ const types = require('./types');
 const bech32 = require('bech32');
 const bs58check = require('bs58check');
 const typeforce = require('typeforce');
+function confidentialAddressToECPubKey(address) {
```
Comment (tiero) [inline src/address.js]: I would call in different way, to make more clear anc consistent: `blindingPubKeyFromConfidentialAddress`

File: test/issuance.spec.ts [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+
+    it('should throw an error if the token amount is < 0', () => {
+      const psbt = createPsbt();
+      const argsInvalidToken = { ...issueArgs, tokenAmount: -2 };
+      assert.throws(() => psbt.addIssuance(argsInvalidToken));
+    });
+
+    it('should throw an error if the asset amount is <= 0', () => {
+      const psbt = createPsbt();
+      const argsInvalidAsset = { ...issueArgs, assetAmount: 0 };
```
Comment (altafan) [inline test/issuance.spec.ts:231]: let's check also negative value

File: test/issuance.spec.ts [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+      const lenOutsAfterIssuance = psbt.data.outputs.length;
+      assert.strictEqual(lenOutsAfterIssuance - lenOutsBeforeIssuance, 1);
+    });
+
+    it('should allow the user to choose the input where to add the issuance', () => {
+      const psbt = createPsbt();
+      psbt.addIssuance(issueArgs, 0);
+      const finalizedPsbt = signAndfinalizeWithAlice(psbt, 0);
+      assert.ok(finalizedPsbt.extractTransaction().ins[0].issuance);
+    });
```
Comment (altafan) [inline test/issuance.spec.ts]: I don't think I got the purpose of this unit test. Can you explain it?

File: test/psbt.spec.ts [hunk 5]
```diff
@@ -740,4 +740,51 @@ describe('Psbt', () => {
   //     assert.ok((psbt as any).data.inputs[index].nonWitnessUtxo.equals(value));
   //   });
   // });
+
+  describe('decode base64 and encode base64', () => {
```
Comment (altafan) [inline test/psbt.spec.ts]: I don't see any assertion for this test. What is this for?

**Pull request merged**